tracker/event_tracker.go

```diff
@@ -34,11 +34,11 @@ func NewEventTracker(
 		rpcEndpoint:  rpcEndpoint,
 		contractAddr: contractAddr,
 		subscriber:   subscriber,
-		logger:       logger,
+		logger:       logger.Named("event_tracker"),
```

Comment:
All event trackers will have the same logger prefix?

Reply:
In fact no. The one instantiated by `StateSyncManager` will have `state-sync` prefix, followed by `event_tracker` suffix. On the other hand, the one from `StateSyncRelayer` is going to have `relayer` prefix (again followed by `event_tracker`). So to me it seems ok. Do you feel the same?